src/EulerIntegration.cpp

```diff
@@ -23,84 +23,140 @@
 #include "MultiBody.h"
 #include "MultiBodyConfig.h"
 
-namespace rbd
-{
-
-Eigen::Matrix<double, 4, 3> angularVelToQuatVel(const std::vector<double>& q)
+namespace
 {
-	return 0.5*(Eigen::Matrix<double, 4, 3>() <<
-		-q[1], -q[2], -q[3],
-		q[0], -q[3], q[2],
-		q[3], q[0], -q[1],
-		-q[2], q[1], q[0]).finished();
-}
+	/** Compute the sum of the first terms of the Magnus expansion of \Omega such that
```

Comment:
Looks strange depending on your tab space setting. Please add a tab in front of the comment start